python/scripts/clean_mes_aides_situation.py

```diff
@@ -0,0 +1,84 @@
+import sys
+import os.path
+import logging
+
+from openfisca_json_customizer import is_json, update_json, delete_key
+
+
+handler = logging.StreamHandler()
+log = logging.getLogger(__name__)
+log.addHandler(handler)
+log.setLevel(logging.DEBUG)
+
+
+def from_mesaides_to_openfisca(json_content):
+  '''
+  Removing mes-aides.gouv.fr application keys from given json content.
+  '''
+
+  delete_key(json_content, u'_id')
+
+  paris_keys = ['paris_complement_sante', 
+    'paris_energie_famille', 'paris_forfait_famille', 
+    'paris_logement', 'paris_logement_familles', 
+    'paris_logement_aspeh', 'paris_logement_plfm', 'paris_logement_psol',
+    'parisien']
+  for key in paris_keys:
+    delete_key(json_content, 'familles/_/' + key)
+  
+  transport_keys = ['brest_metropole_transport', 'rennes_metropole_transport']
+  for key in transport_keys:
+    delete_key(json_content, 'individus/demandeur/' + key)
+    delete_key(json_content, 'individus/enfant_0/' + key)
+
+  delete_mesaides_pre_computed_keys(json_content, '2018-01')
```

Comment:
`2018-01`should be taken out of the function  call and put in an easily findable variable